tests/test_dbapi.py

```diff
@@ -178,16 +168,48 @@ def __fspath__(self):
         with monetdbe.connect(path) as cx:
             cx.execute('create table test(id integer)')
 
-    @unittest.skip("TODO: Not yet implemented, see issue #22")
-    def test_OpenUri(self):
-        self.addCleanup(rmtree, TESTFN)
-        with monetdbe.connect(TESTFN) as cx:
-            cx.execute('create table test(id integer)')
-        with monetdbe.connect('file:' + TESTFN, uri=True) as cx:
-            cx.execute('insert into test(id) values(0)')
-        with monetdbe.connect('file:' + TESTFN + '?mode=ro', uri=True) as cx:
+    def test_InTransaction(self):
```

Comment:
looks like you changed the order of these tests which messes with the github pr diff.